src/utils.cpp

```diff
@@ -80,4 +81,12 @@ string drillUp(const vector<string>& targets, const string& fail, const string&
 	}
 	return {};
 }
+
+const char* newCString(std::string value) {
```

Comment:
is this different from `strdup()`?

Reply:
You're right. I changed it to using `strdup()`.